parsers/IN_WE.py

```diff
@@ -93,17 +98,46 @@ def fetch_data(
                 item[datetime_col] = dt.shift(minutes=1).floor("minute").datetime
             else:
                 item[datetime_col] = dt.floor("minute").datetime
-        df_data = pd.DataFrame(
-            [item for item in data if item[datetime_col].hour == dt_12_hour.hour]
-        )
-        return df_data
+        return data
```

Comment:
Are we still returning a data frame ? For me it looks like we're returning a dictionary. Either we should update the signature of the function or make sure to return a data frame. Besides I think that now all the code above is not used anymore because we return data. All the operations done on items do not seem to have an effect?
```
        return data
```